package.json

```diff
@@ -41,12 +41,15 @@
 		"@lezer/javascript": "^1.4.5",
 		"@lezer/lr": "^1.3.9",
 		"@neocodemirror/svelte": "^0.0.17",
+		"@open-rpc/client-js": "^1.8.1",
 		"@replit/codemirror-lang-svelte": "^6.0.0",
 		"@replit/codemirror-vim": "^6.0.14",
 		"@sveltejs/site-kit": "^3.2.2",
 		"@webcontainer/api": "^1.1.5",
 		"@yaireo/relative-time": "^1.0.4",
 		"as-comps": "^0.0.27",
+		"codemirror-languageserver": "^1.11.0",
+		"events": "^3.3.0",
```

Comment:
are these new modules (open-rpc, codemirror-languageserver, events) used anywhere? 👀

Reply:
removed open-rpc, and events. since they were required when using the language server modules directly rather than using the current NPM package. `codemirror-languageserver` is used here, though: https://github.com/SvelteLab/SvelteLab/pull/277/files#diff-eb2ab2a29594e8de109dcc59d1e2664eb6b05c0107da708a5119794aef7f030aR275